{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/console/Exception/InvalidArgumentException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Console\\Exception","location":[11,0,11,46]},{"kind":1,"name":"Symfony\\Component\\Console\\Exception\\InvalidArgumentException","location":[16,0,18,1],"associated":[{"kind":1,"name":"InvalidArgumentException"},{"kind":2,"name":"Symfony\\Component\\Console\\Exception\\ExceptionInterface"}]}]}}